coalib/bears/requirements/PackageRequirement.py

```diff
@@ -0,0 +1,44 @@
+from coala_decorators.decorators import generate_eq, generate_repr
+
+
+@generate_eq("manager", "package", "version")
+@generate_repr()
+class PackageRequirement:
+    """
+    This class helps keeping track of bear requirements. It should simply
+    be appended to the REQUIREMENTS tuple inside the Bear class.
+
+    Two ``PackageRequirements`` should always be equal if they have the same
+    manager, package and version:
+
+    >>> pr1 = PackageRequirement('pip', 'coala_decorators', '0.1.0')
+    >>> pr2 = PackageRequirement('pip', 'coala_decorators', '0.1.0')
+    >>> pr1 == pr2
+    True
+    """
+
+    def __init__(self, manager: str, package: str, version=""):
+        """
+        Constructs a new PackageRequirement.
+
+        :param manager: A string with the name of the manager (pip, npm, etc).
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version. Leave empty to
+                        install the latest.
```

Comment:
I think you should not write "number", as this suggerates an integral type. Maybe just: "A version string. Leave empty to specify latest version." (also not that I rephrased the last part, because you don't necessary install stuff with this class, essentially this just defines a requirement^^)